templates/files/apps/common/cilium-app.yaml

```diff
@@ -96,4 +99,4 @@ spec:
     secret:
       name: ""
       namespace: ""
-  version: 0.12.0
+  version: 0.21.0
```

Comment:
Is it intentional that we upgrade Cilium (v1.13.x to v1.14.x)?

Reply:
yeah